spec/routing/checksum_records_routing_spec.rb

```diff
@@ -1,30 +1,27 @@
-require "rails_helper"
+require 'rails_helper'
 
 RSpec.describe ChecksumRecordsController, type: :routing do
-  describe "routing" do
+  describe 'routing' do
 
-    it "routes to #index" do
-      expect(get: "/checksum_records").to route_to("checksum_records#index")
+    it 'routes to #show via GET' do
+      expect(get: '/checksum_records/1').to route_to('checksum_records#show', id: '1')
     end
 
-    it "routes to #show" do
-      expect(get: "/checksum_records/1").to route_to("checksum_records#show", id: "1")
+    it 'routes to #create via POST' do
+      expect(post: '/checksum_records').to route_to('checksum_records#create')
     end
 
-    it "routes to #create" do
-      expect(post: "/checksum_records").to route_to("checksum_records#create")
+    it 'does not route to #update' do
+      expect(patch: '/checksum_records').not_to be_routable
+      expect(put: '/checksum_records').not_to be_routable
```

Comment:
seems like this is causing the rubocop error that's failing the build.  i would personally be fine with allowing multiple expectations per test, but splitting this into two tests would also be fine by me.